lib/rucio/web/ui/common/saml/advanced_settings.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
this will have to be customised for the CERN SAML provider, right?

Reply:
Yes, the entity Id and single sign-on service url need to be customized as per CERN SAML provider

Reply:
Should we rename this specific one then to template?